src/scripts.js

```diff
@@ -1,10 +1,15 @@
 import $ from 'jquery';
-import users from './data/users-data';
-import recipeData from  './data/recipe-data';
-import ingredientData from './data/ingredient-data';
+// import users from './data/users-data';
+// import recipeData from  './data/recipe-data';
+// import ingredientData from './data/ingredient-data';
 
 import './css/base.scss';
 import './css/styles.scss';
+import './images/seasoning.png';
```

Comment:
Glad we got these images in there! They look great!

Reply:
I concur.